docs/balances/FAQs.md

```diff
@@ -8,6 +8,50 @@ tags: [Balances]
 
 Valuations provide a history of the value of a portfolio whereas balances provides an indicative value of a portfolio currently, based on the previous day's closing prices for securities and FX rates.
 
+Valuations are created at approximately the same time every day, for the previous day. Transactions booked with a date of `t` are only reflected in the valuation for `t+1`.
```

Comment:
```suggestion
Valuations are dated, and only transactions with a timestamp on or before a valuation's date will be reflected in the valuation.
```

This level of detail around when valuations are created etc. should be in a Valuations section of the docs IMO.